src/auth/project-config.ts

```diff
@@ -57,13 +66,19 @@ export class ProjectConfig {
    * This is based on the calling code of the destination phone number.
    */
   public readonly smsRegionConfig?: SmsRegionConfig;
+  /**
+   * The Multi Factor Config for the project.
+   * Configures the multi factor settings for the project.
+   * Supports only phone and TOTP
+   */
+  public readonly multiFactorConfig?: MultiFactorAuthConfig;
 
   /**
    * Validates a project config options object. Throws an error on failure.
    *
    * @param request - The project config options object to validate.
    */
-  private static validate(request: ProjectConfigClientRequest): void {
+  private static validate(request: any): void {
```

Comment:
It is better if we can keep the strong type here. Can we stick to `ProjectConfigClientRequest` over `any`?

Reply:
Because of the inconsistent naming of types between backend and SDK, we can't keep a strong type to have flexibility in converting. The same thing has been done in tenant.ts : ```private static validate(request: any, createRequest: boolean): void {```

Reply:
Reopening since this still shows the "any" type. Can we use the UpdateProjectConfigRequest type here?

Reply:
This somehow seems to be getting auto-resolved. @pragatimodi can you look into making this type change?